robot/EDA/tests/browser/settings/affiliations.robot

```diff
@@ -0,0 +1,47 @@
+*** Settings ***
+Documentation   Validate relationships tab in EDA Settings
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/AffiliationsSettingsPageObject.py
+...             robot/EDA/resources/SystemSettingsPageObject.py
+
+Suite Setup     Open Test Browser
+Suite Teardown  Capture screenshot and delete records and close browser
+
+Test Setup      Go to EDA settings tab          Affiliations
+
+*** Test Cases ***
+Verify checkbox values can retain values on save
+    [Documentation]         Updates the checkbox values of affiliations settings tab and verifies
+    ...                     the updated values are retained upon save
+    [tags]                    unstable        W-8177333     rbt:high
+    Update checkbox value
+    ...         Enable Record Type Validation=true
```

Comment:
So this keyword internally takes care of going into edit mode, update the checkbox and save ?

Reply:
Yes! It goes into edit mode and makes updates (only when necessary and saves the changes)